tests/feature_elimination/test_feature_elimination.py

```diff
@@ -322,8 +323,47 @@ def test_shap_rfe_early_stopping(complex_data, capsys):
     assert len(out) == 0
 
 
+def test_shap_rfe_early_stopping_CatBoost(complex_data, capsys):
```

Comment:
Can you also add one test for XGBoost, just to make sure it works as well ?

Reply:
Added, something a bit weird happened tho.

In the last test XGBoost gets a test score of 0.3333.